src/app/modules/ix-forms/components/ix-input/ix-input.component.ts

```diff
@@ -66,6 +71,10 @@ export class IxInputComponent implements ControlValueAccessor {
     this.onChange(this.value);
   }
 
+  inValidMessage(): string {
```

Comment:
The spelling looks weird. Could we have `invalid` and `invalidMessage`?

Reply:
Sure!